udata/tests/plugin.py

```diff
@@ -43,14 +43,18 @@ def delete(self, url, data=None, **kwargs):
     def login(self, user=None):
         user = user or UserFactory()
         with self.session_transaction() as session:
-            session['user_id'] = str(user.id)
+            # Since flask-security-too 4.0.0, the user.fs_uniquifier is used instead of user.id for auth
+            user_id = getattr(user, current_app.login_manager.id_attribute)()
```

Comment:
The `login_manager.id_attribute` returns the correct attribute depending on the flask-security version. It now returns the `user.fs_uniquifier`.